src/lib/expansion-panel/expansion-panel-core.ts

```diff
@@ -110,4 +137,16 @@ export class ExpansionPanelCore implements IExpansionPanelCore {
       this._adapter.setHostAttribute(EXPANSION_PANEL_CONSTANTS.attributes.ANIMATION_TYPE, this._animationType);
     }
   }
+
+  public get trigger(): string {
+    return this._trigger;
+  }
+  public set trigger(value: string) {
+    if (this._trigger !== value) {
+      this._adapter.detachTriggerAria();
+      this._trigger = value;
+      this._syncTrigger();
+      this._adapter.setHostAttribute(EXPANSION_PANEL_CONSTANTS.attributes.TRIGGER, this._trigger);
```

Comment:
We're starting to get away from reflecting properties to attributes actually. Let's go ahead and remove this call to `setHostAttribute()` so that we don't have to remove it later when we do a sweep across the library.